content/en/docs/reference/command-line-tools-reference/feature-gates/InformerResourceVersion.md

```diff
@@ -0,0 +1,15 @@
+---
+title: InformerResourceVersion
+content_type: feature_gate
+
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha 
+    defaultValue: false
+    fromVersion: "1.30"
+---
+Enables the check over the last synced resource version using the informer. 
```

Comment:
I don't know what this means. I could look it up, but we should try to write a description that people can understand without looking elsewhere.

Reply:
https://github.com/kubernetes/kubernetes/pull/123344#issuecomment-2258456017